packages/web/config/ibc-assets.ts

```diff
@@ -156,19 +156,15 @@ export const IBCAssetInfos: (IBCAsset & {
         },
       },
       {
-        counterpartyChainId: "axelar-dojo-1",
-        sourceChannelId: "channel-208",
-        destChannelId: "channel-3",
-        coinMinimalDenom: "uusdt",
-        sourceChainNameOverride: "Ethereum",
+        counterpartyChainId: "kava_2222-10",
+        sourceChannelId: "channel-143",
+        destChannelId: "channel-1",
+        coinMinimalDenom: "erc20/tether/usdt",
         isVerified: true,
-        originBridgeInfo: {
-          bridge: "axelar" as const,
-          wallets: ["metamask" as const, "walletconnect" as const],
-          method: "deposit-address" as const,
-          sourceChainTokens: [AxelarSourceChainConfigs.usdt.ethereum],
-        },
+        //depositUrlOverride: "https://app.kava.io/transfer",
```

Comment:
same question. Why is this commented out?

Reply:
Normal ibc transfers just go to IBC chains, and that's what we're doing for now (because it's commented out), but that only goes to Kava IBC, not Kava EVM (used by CEXes). The commented out URL is Kava's transfer page to turn USDT on Kava IBC to USDT on Kava EVM, but it's geoblocked. So we'll soon want override these Deposit and Withdraw links to redirect to some other page that can do that transfer (maybe this one once geoblock is removed).